### PR TITLE
feat: replace namespace of configMaps and roleBindings

### PR DESCRIPTION
**Pull request by ksimon1**
**What this PR does / why we need it**:
feat: replace namespace of configMaps and roleBindings

When ssp CR contains defined spec.tektonPipelines.namespace attribute, it should replace kubevirt.io/deploy-namespace annotation (currently used only for configMaps and roleBindings in example tekton pipelines). So when user decides to deploy pipelines to different namespace, the pipeline will be able to pick up correct configMap and roleBinding.

**Which issue(s) this PR fixes**: 
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=2229903

**Release note**:
```
feat: replace namespace of configMaps and roleBindings in tekton pipelines
```

### CODE REVIEW

File: internal/operands/tekton-pipelines/reconcile.go [hunk 1]
```diff
@@ -89,7 +89,7 @@ func (t *tektonPipelines) Reconcile(request *common.Request) ([]common.Reconcile
 		return nil, nil
 	}
 	if !request.CrdList.CrdExists(tektonCrd) {
-		return nil, fmt.Errorf("Tekton CRD %s does not exist", tektonCrd)
+		return nil, fmt.Errorf("tekton CRD %s does not exist", tektonCrd)
```
Comment (0xFelix) [inline internal/operands/tekton-pipelines/reconcile.go:92]: nit: necessary change?

File: internal/operands/tekton-pipelines/reconcile.go [hunk 2]
```diff
@@ -180,13 +180,13 @@ func reconcileTektonPipelinesFuncs(pipelines []pipeline.Pipeline) []common.Recon
 
 func reconcileConfigMapsFuncs(configMaps []v1.ConfigMap) []common.ReconcileFunc {
 	funcs := make([]common.ReconcileFunc, 0, len(configMaps))
+	var userDefinedNamespace bool
 	for i := range configMaps {
 		configMap := &configMaps[i]
 		funcs = append(funcs, func(request *common.Request) (common.ReconcileResult, error) {
-			if value, ok := configMap.Annotations[deployNamespaceAnnotation]; ok {
+			configMap.Namespace, userDefinedNamespace = getTektonPipelinesNamespace(request)
+			if value, ok := configMap.Annotations[deployNamespaceAnnotation]; ok && !userDefinedNamespace {
```
Comment (0xFelix) [inline internal/operands/tekton-pipelines/reconcile.go:188]: Is this required to keep compatibility with namespace annotations?

**Comment by jcanocan** [comment]
This PR is blocked by CI issue. IMHO we can force merge it without tests.

**Comment by lyarwood** [comment]
Agreed we can /override here.

**Pull request merged**